src/rcc.rs

```diff
@@ -237,6 +272,20 @@ impl CFGR {
 
         let rcc = unsafe { &*RCC::ptr() };
 
+        if self.backup_domain.is_some() {
+            // Configure the perscaler to use the LSE clock as defined in the documentation
+            // in section 7.2.4. This gives a 32.768khz frequency for the RTC
+            rcc.bdcr.modify(|_, w| {
+                w
+                    // Enable external low speed oscilator
+                    .lseon().set_bit()
+                    // Enable the RTC
+                    .rtcen().set_bit()
+                    // Set the source of the RTC to LSE
+                    .rtcsel().lse()
+            });
```

Comment:
I'd prefer to have a `pub fn lse<F>(self, freq: F) -> Self` in CFGR (and a convenient `pub fn lse_is_32k(self)`), and then start it here and note the freq in `Clocks`

Reply:
Out of date because of my comment